### ISSUE TITLE
Feature request: option to customize the notification time individualy

### ISSUE THREAD

**Issue by moshpirit** [Etar-Group/Etar-Calendar#314]
There's no option for example to stablish 4 hours before the event. It would be nice to have a customized clock view to determine the amount of time before the event to notify it.

**Comment by apogarte** [comment]
Hi
Customizing time of reminder would be great ... i miss this too ...
Manu thanks anyways ! 😊

**Comment by apogarte** [comment]
+++ !!!
 Thanks !!!

**Comment by jhoc2000** [comment]
+!
A "custom" time before the event using the date and time pickers would make this app "excellent" for me.

**Comment by jwhaleyca** [comment]
Agreed. I usually set a day of reminder at 6 am or 9 am as I rarely remember from an alert the day before. I may block out a whole day for a specific reason and remind myself that morning. This feature is critical for busy people. I do love the app for its simplicity. Nothing comes close that is as comprehensive in features, let alone a CalDAV calendar.

**Comment by RizzoV** [comment]
Was going to open a new issue when I discovered this one already exists, this would be an extremely welcomed feature IMHO.

Sadly, I'm currently using Simple Calendar for this sole feature.

**Comment by IlToro** [comment]
This calendar should definitely have this feature and the possibility to add a notification at the end of the event (so that you can use it as a timebox as well)!

**Comment by elig0n** [comment]
why is there 2 days before event notification time but not 1 day?

**Comment by jhoc2000** [comment]
24 hours is one day. 

However, it would be much better if there were a custom notification time /period.

**Comment by apogarte** [comment]
Hi
I would  be glad to be able to choose any period like 23 hours, 17 hours, something more open... Than a closed restricted choice like by nos


Envoyé depuis mon HTC

----- Message de réponse -----
De : "jhoc2000" <notifications@github.com>
Pour : "Etar-Group/Etar-Calendar" <Etar-Calendar@noreply.github.com>
Cc : "apogarte" <apogarte@gmx.fr>, "Comment" <comment@noreply.github.com>
Objet : [Etar-Group/Etar-Calendar] Feature request: option to customize the notification time individualy (#314)
Date : dim., févr. 28, 2021 10:01

24 hours is one day. 



However, it would be much better if there were a custom notification time /period. 



On 28 February 2021 5:07:13 pm AEDT, elig0n <notifications@github.com> wrote:

>why is there 2 days before event notification time but not 1 day? 



&mdash;You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Etar-Group/Etar-Calendar/issues/314#issuecomment-787419552",
"url": "https://github.com/Etar-Group/Etar-Calendar/issues/314#issuecomment-787419552",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by Hubert21** [comment]
You cannot set a reminder during daytime that you ll have a train to take at 4 am in 3 days

**Comment by Hubert21** [comment]
You can not set a notification during daytime that you'll have a train to take at 3am in 3 days

**Comment by llevrel** [comment]
This is a highly desirable feature! Replacing the radio button list by, say, a box to type a number and a radio button list for minutes/hours/days (like what Evolution offers). If one wants non-integer number of hours or days they can still use minutes ou hours (e.g. 36 h for one and a half day).

**Comment by jhoc2000** [comment]
I totally agree. 

The number of times that I've wanted notification an hour and three quarters or two and a half hours before a meeting has prompted me to switch to another calendar app.

**Comment by apogarte** [comment]
Looks very promising. Just a box to type the time directly...

Le 10 septembre 2022 01:14:18 GMT+02:00, jhoc2000 ***@***.***> a écrit :
>I totally agree. 
>
>The number of times that I've wanted notification an hour and three quarters or two and a half hours before a meeting has prompted me to switch to another calendar app.
>
>On 10 September 2022 1:20:42 am AEST, llevrel ***@***.***> wrote:
>>This is a highly desirable feature! Replacing the radio button list by, say, a box to type a number and a radio button list for minutes/hours/days (like what Evolution offers). If one wants non-integer number of hours or days they can still use minutes ou hours (e.g. 36 h for one and a half day).
>>
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/Etar-Group/Etar-Calendar/issues/314#issuecomment-1242548690
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
-- Envoyé depuis /e/ Mail.
